plugins/slack_webhooks/app/models/slack_webhook_notification.rb

```diff
@@ -1,14 +1,17 @@
 class SlackWebhookNotification
-  def initialize(deploy)
+  def initialize(deploy, deploy_phase=:for_buddy)
     @deploy = deploy
+    @deploy_phase = deploy_phase
     @stage = deploy.stage
     @project = @stage.project
     @user = @deploy.user
   end
 
-  def deliver
+  def deliver(message=content)
```

Comment:
Tried to tweak it but gave up. The default value `content()` is for deployment notification message, which was computed in this class. We only want to explicit specify the `message` for buddy request.